### PR TITLE
BIC-4483 styling complete

### PR DESCRIPTION
**Pull request by AlejandroHCarrillo**

### CODE REVIEW

**Comment by jgalvanbam** [comment]
Please remove this control from this commit. this will be done in a separated ticket.
https://github.com/bambora-na/dev.na.bambora.com/pull/155/files#diff-90618c41db7e37ea0b7b17edb2eef1d9bda23cb1f4352fd723a695299754e998R45

File: source/docs/forms/create_test_merchant_account.html.erb [hunk 1]
```diff
@@ -38,6 +42,19 @@ breadcrumbs: false
   <%= text_field_tag :company_login, :placeholder => 'Your company', :id => 'company_login', :required => true %>
   </div>
 
+  <div class="form-group">
```
Comment (jgalvanbam) [inline source/docs/forms/create_test_merchant_account.html.erb]: Remove this control. this will be done in a separated ticket.

**Review by jgalvanbam — CHANGES_REQUESTED** [review]

File: source/docs/forms/create_test_merchant_account.html.erb (hunk 1 above)
Comment (AlejandroHCarrillo) [reply source/docs/forms/create_test_merchant_account.html.erb]: The Kind of business combo code was removed

File: source/javascripts/na/create_test_merchant_account.js [hunk 2]
```diff
@@ -93,16 +95,37 @@ $(function() {
...
+
     if(getUrlParameter('status')) {  // (i.e. if there is a response in the url)
-       
+        var combovalue = getUrlParameter('kind_of_business');
+        console.log("kind_of_business combovalue: ", combovalue);
+   
         var $currentForm = $('#createTestAccount_form');
         var $statusDiv = $currentForm.next('.block-highlight');
-        var $statusParagraph = $statusDiv.find('p');         
+        var $statusParagraph = $statusDiv.find('p');
```
Comment (jgalvanbam) [inline source/javascripts/na/create_test_merchant_account.js:97]: Remove this message, this is part of the dropdown control removed.

File: source/javascripts/na/create_test_merchant_account.js [hunk 3]
```diff
@@ -138,5 +163,27 @@ $(function() {
             $statusDiv.removeClass('hidden success notice');
             $statusDiv.addClass('error');
         }
-    } 
+    }
+    
+    window.onload = function() {
```
Comment (jgalvanbam) [inline source/javascripts/na/create_test_merchant_account.js]: Also with these lines, are these part of the removed control?

**Review by jgalvanbam — CHANGES_REQUESTED** [review]
Changes needed

File: source/javascripts/na/create_test_merchant_account.js (hunk 2 above)
Comment (AlejandroHCarrillo) [reply source/javascripts/na/create_test_merchant_account.js:97]: Done!

File: source/javascripts/na/create_test_merchant_account.js (hunk 3 above)
Comment (AlejandroHCarrillo) [reply source/javascripts/na/create_test_merchant_account.js]: Done!

**Comment by jgalvanbam** [comment]
This PR will be in conflict with this other one https://github.com/bambora-na/dev.na.bambora.com/pull/156 . We will close this and create an new one based on latest changes.

**Pull request closed without merging**